server/models/experience_model.py

```diff
@@ -0,0 +1,23 @@
+from extensions import db
+
+
+class ExperienceModel(db.Model):
+    __tablename__ = "experience"
+
+    id = db.Column(db.Integer, primary_key=True)
+    user_id = db.Column(db.Integer, db.ForeignKey(
+        "users.id"), nullable=False)
+    language = db.Column(db.String(20), nullable=False)
```

Comment:
I wanted you to add a enum of possible languages here so that when you add experience you make sure it is properly spelled

Reply:
added Language enum class, and added checks against this enum when user_experience api and request_new api is called.